chain/laika/CHANGELOG.rst

```diff
@@ -0,0 +1,12 @@
+==========
+Change Log
+==========
+
+1.0.0 (2020-03-24)
+-------------------------------
+- Add fork to new validator set as fixed static JSON array
+- Add a second fork to a validator set as contract 100 blocks later
+- Both forks include the same 5 addresses, controlled by the Trustlines
```

Comment:
This and the next one are not own new additions, but only explanation to the first two changes. So please add them there and not as an own point.

Reply:
:heavy_check_mark: